src/DTO/Request/ISDSSearch3.php

```diff
@@ -41,7 +41,7 @@ class ISDSSearch3 implements IRequest
      * @Serializer\XmlElement(cdata=false)
      * @Serializer\SerializedName("p:page")
      */
-    private int $page = 1;
+    private int $page = 0;
```

Comment:
Touto změnou si nejsem na 100% jistý. Mrknu na dokumentaci

Reply:
Jj, opravdu je to tak. 
![image](https://user-images.githubusercontent.com/4020490/154819194-3a00799f-5f62-4e24-8eea-e52d055a863a.png)